### DISCUSSION TITLE
New device request processed and closed but device not in the supported devices

### DISCUSSION THREAD

**Discussion by JackJackattack4** [make-all/tuya-local#1777] [Q&A] [answered]
Couple of months ago, I opened a new device request and it was closed after being what I believe "processed". 

I might have missed it, but I can't find the implementation of the deakon heatpump device in the local_tuya update, I don't see it. Was it renamed?

Details of this available at https://github.com/make-all/tuya-local/issues/1288#issuecomment-1876047460_

**Comment by make-all** [comment, marked as answer]
The report was closed because it did not contain the requested information.
